.github/workflows/release-node.yml

```diff
@@ -0,0 +1,31 @@
+# This action triggers a GitLab CI job that generates the release notes
+name: Node Release
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'main'
+
+jobs:
+  release_check:
+#    This job will only run if:
+#     * the pull request is closed and merged to main branch;
+#     * the pull request has the "release:" and "node" in its title
+    if: ${{ github.event.pull_request.merged == true && contains(github.event.pull_request.title, 'release:') && contains(github.event.pull_request.title, 'node')  }}
```

Comment:
We would need to make sure that the PRs now include both `release:` and `node`. The PRs you automatically create in the version-up workflow don't do this. How about we change the condition here to check for `release node: ....` instead of checking for both words separately and then we make the version-up workflow create the PR title `release node: ` prefixed?

Reply:
edited